### PR TITLE
Reformat Check-in files

### PR DESCRIPTION
**Pull request by nikosev**
# Summary

Reformat Check-in files using VScode's Prettier and markdownlint extensions

### CODE REVIEW

**Review by gwarf — CHANGES_REQUESTED** [review]
Thanks @nikosev, so quickly looking it seems it's still missing mainly addressing the long lines due to the big tables.
For this we advise two things:
- moving to something else than a table when a table doesn't really makes sense
- when a table is indeed the proper way to represent this data, it can be encapsulated with some tags to disable markdown-lint checks:

```markdown
<!-- markdownlint-disable line-length -->
| Action           | rOCCI                                                                                                                | OpenStack                                                                                |
| ---------------- | -------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------- |
| List images      | `occi -a list -r os_tpl`                                                                                             | `openstack image list`                                                                   |
<!-- markdownlint-enable line-length -->
```

File: content/en/providers/check-in/sp/_index.md [hunk 1]
```diff
@@ -180,103 +163,92 @@ Service Providers is included in the [User Attribute](#user-attributes) section.
...
 
-- [Shibboleth SP
-  Configuration](https://wiki.shibboleth.net/confluence/display/SHIB2/NativeSPConfiguration)
-- [SimpleSAMLphp Service Provider
-  QuickStart](https://simplesamlphp.org/docs/stable/simplesamlphp-sp)
-- [Simple SAML 2.0 service provider with mod_auth_mellon Apache
-   module](https://github.com/UNINETT/mod_auth_mellon)
+- [Shibboleth SP Configuration](https://wiki.shibboleth.net/confluence/display/SHIB2/NativeSPConfiguration)
+- [SimpleSAMLphp Service Provider QuickStart](https://simplesamlphp.org/docs/stable/simplesamlphp-sp)
+- [Simple SAML 2.0 service provider with mod_auth_mellon Apache module](https://github.com/UNINETT/mod_auth_mellon)
```
Comment (nikosev) [inline content/en/providers/check-in/sp/_index.md:181]: @gwarf should I disable the line length check for the links too?
for example in these lines

File: content/en/users/check-in/voms/_index.md [hunk 2]
```diff
@@ -252,34 +259,33 @@ existing VOs
...
 
-``` {.console}
+```{.console}
 voms-proxy-init --voms <name of the vo> --rfc
 ```
 
 See for example, using `fedcloud.egi.eu` VO:
 
-``` {.console}
+```{.console}
```
Comment (gwarf) [inline content/en/users/check-in/voms/_index.md:288]: Please use proper markdown fenced code markers (everywhere in the doc), like;
```
```sh
```

(sh is usually preferred over console as it highlights more stuff).

File: content/en/providers/check-in/sp/_index.md (hunk 1 above)
Comment (gwarf) [reply content/en/providers/check-in/sp/_index.md:181]: so I think/hope that for links it shouldn't be a problem as we are not expected to shorten them/split them over lines, but let me double check.

**Comment by gwarf** [comment]
You may also have to also disable `no-inline-html` for the HTML code that was added into tables.

File: content/en/providers/check-in/sp/_index.md (hunk 1 above)
Comment (gwarf) [reply content/en/providers/check-in/sp/_index.md:181]: We will see how it works once the tables will have been fixed, I expect it's the main problem.

**Comment by nikosev** [comment]
Thank you @gwarf for your comments. I have resolved the errors!

File: content/en/providers/check-in/sp/_index.md (hunk 1 above)
Comment (gwarf) [reply content/en/providers/check-in/sp/_index.md:187]: Great, so all seems fine with long URL.

**Review by gwarf — APPROVED** [review]

**Comment by gwarf** [comment]
Perfect, thanks!
Let's get this merged.

**Pull request merged**